src/index.ts

```diff
@@ -62,11 +62,20 @@ const fileTypesMap: Record<string, string> = {
 
 const files = await Promise.all(filePaths.map(async filePath => {
     const type = fileTypesMap[path.extname(filePath)];
+    // Check if the path is an url
+    if (/^https?:\/\//.test(filePath)) {
+        const url = new URL(filePath);
+        const res = await fetch(url);
+        const data = await res.blob();
+
+        return new File([data], path.basename(url.pathname));
```

Comment:
need to copy the type to the file
```suggestion
        return new File([data], path.basename(url.pathname), {type: data.type});
```